students/rgpag/lesson04/trigrams.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+import random
+
+# txt file to be used
+text_in = 'sherlock_full.txt'
+f = open(text_in)
+msg = f.read()
+f.close
+
+# manipulate text file to be more trigram friendly
+string = msg.lower()
+string = string.replace("\n", " ")
+string = string.replace("--", " ")
+string = string.replace("-", " ")
+spl_string = string.split(" ")
+
+# init vals
+str_dict = {}
+i = 0
+j = 0
+quit = 0
```

Comment:
Overwriting built-in `quit` here, also it's good to define a flag variable like this closer to the point in code at which you are using it